### PR TITLE
Remove unused window.tagit_opts declarations

### PR DESCRIPTION
**Pull request by lb-**
- Additional clean up from #10102
- See #10100
- The window global is no longer used, widgets come with the base tags URL by default now using Stimulus data attributes

### CODE REVIEW

**Comment by lb-** [comment]
@laymonage - some additional clean up I missed on #10102 - could you please take a look.

If you want to manual re-test, here are the places to look (check that the tag widget is functional).

* http://localhost:8000/admin/pages/62/edit/ (add an image with the chooser
* http://localhost:8000/admin/images/multiple/add/ add multiple images
* http://localhost:8000/admin/images/add/ add single image
* localhost:8000/admin/documents/multiple/add/ add multiple documents
* http://localhost:8000/admin/documents/add/ add single document

I have just smoked tested all of these after these change but do not want to merge without another approval.

**Review by laymonage — APPROVED** [review]
Oh, didn't know we had such configuration as well. Yep, tested all the views and tagging (with autocomplete) still works. Searched for `tagit` in the codebase and I believe the remaining ones are the correct usage and the library itself, so I think we're good. Thanks!

**Pull request merged**
